docs/README.md

```diff
@@ -27,39 +60,47 @@ The Product Customizer allows a product's [attachments](https://help.vtex.com/tu
   }
 ```
 
+### `assembly-option-input-values` props
+
 | Prop name | Type | Description | Default value |
 |--------------|--------|--------------| --------|
-| `optionsDisplay` | String | You should choose between “box” or “select”. This will define whether the attachment's pre-defined options will be displayed to be selected in a Checkbox (`box`) or a dropdown list (`select`) . | `select` | 
+| `optionsDisplay` | `enum` | Define whether the attachment's pre-defined options will be displayed to be selected in a Checkbox (`box`) or in a dropdown list (`select`) . | `select` | 
+
+### `assembly-option-item-customize`props
+
+| Prop name | Type | Description | Default value |
+|--------------|--------|--------------| --------|
+| `buttonProps` | `enum` | ???????. Possible values are: `left`, `right` or `none`. | ?? | 
```

Comment:
buttonProps is an object with:

`{ collapse: 'left' | 'right' }`

There's no default value.

Also it accepts `children`, its children it's the content of the modal that will be rendered once the user clicks in the Customize button.

![image](https://user-images.githubusercontent.com/284515/85619662-00bd0480-b639-11ea-9a22-28dd02f1a1a4.png)

Once clicked:
![image](https://user-images.githubusercontent.com/284515/85619698-09add600-b639-11ea-816c-7bc819163717.png)

Example:
https://github.com/vtex-apps/store-theme/blob/master/store/blocks/product-assembly.jsonc#L59-L115